helper-cli/src/main/kotlin/HelperMain.kt

```diff
@@ -100,3 +111,28 @@ internal class HelperMain : CliktCommand(name = ORTH_NAME, epilog = "* denotes r
         printStackTrace = stacktrace
     }
 }
+
+/**
+ * Read [ortFile] into an [OrtResult] and return it.
+ */
+fun CliktCommand.readOrtResult(ortFile: File): OrtResult {
+    val (ortResult, duration) = measureTimedValue { ortFile.readValue<OrtResult>() }
+
+    log.perf {
```

Comment:
I think performance logging is not needed in the helper cli and prefer to not add it.

Reply:
If the last renaming `log` would go away, then this would indeed not need to be an extension function anymore.

Reply:
But does it hurt to have optional performance logging? It's only visible if the debug flag is set.

Reply:
...or the `--perf` flag 😉 Speaking of performance (for *serializing* result files, though), did you see an drop in your number since https://github.com/oss-review-toolkit/ort/pull/3999/commits/738834057e41e99e6a44255fb322dd109f725c1d was merged?

Reply:
> But does it hurt to have optional performance logging? 

I believe that's the wrong question to ask, because unneeded code IMO should be avoided.

Reply:
I think "unneeded" is a matter of opinion in this case. However, I have removed it to not prolong the discussion.

Reply:
> ...or the `--perf` flag wink Speaking of performance (for _serializing_ result files, though), did you see an drop in your number since [7388340](https://github.com/oss-review-toolkit/ort/commit/738834057e41e99e6a44255fb322dd109f725c1d) was merged?

The duration went down a bit, but due to the different result sizes it's difficult to draw conclusions. The deserialization:serialization ratio went roughly from 4:1 to 5:1, but that could also be coincidence or caused by other factors. I think the best way to check the impact would be a before/after comparison for a large file.